go/integTest/cluster_commands_test.go

```diff
@@ -1424,6 +1424,16 @@ func (suite *GlideTestSuite) TestClientSetGetNameWithRoute() {
 	}
 }
 
+func (suite *GlideTestSuite) TestMoveCluster() {
+	client := suite.defaultClusterClient()
+	t := suite.T()
+	key := uuid.New().String()
+	suite.verifyOK(client.Set(context.Background(), key, "hello"))
+	result, err := client.Move(context.Background(), key, 2)
```

Comment:
This is going to fail when we attempt to run in the pipeline which tests against various server versions. We either need logic in the command to gracefully handle running in Valkey 7 and 8 or we need to add logic in the test to skip the test when running before Valkey 9.

Reply:
Added Skip if the server version is lower the 9